CHANGES.txt

```diff
@@ -1,5 +1,7 @@
 Current
 
+Fixed: GITHUB-2913: Maps containing nulls can be incorrectly considered equal
```

Comment:
@ahgittin It would be good if you could please help add your ID/name at the end of this line, so that we can call out your contribution in the release notes of this version (when it goes out). Its our small way of thanking you for helping TestNG become better.